### PR TITLE
[CUBRIDMAN-183] Remove CUBRIDDriver.getDefaultConnection()

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CUBRIDMAN-183

We don't support CUBRIDDriver.getDefaultConnection() anymore.

### CODE REVIEW

**Review by mhoh3963 — APPROVED** [review]

File: en/sql/jsp.rst [hunk 1]
```diff
@@ -331,18 +331,12 @@ Creating Connection
 ---------------------
 
 To access the database from a Java stored function/procedure, you must use the server-side JDBC driver.
-To acquire a connection to the database using the server-side JDBC driver, you can either use "**jdbc:default:connection:**" as the URL for JDBC connection, or call the **getDefaultConnection** () method of the **cubrid.jdbc.driver.CUBRIDDriver** class.
+To acquire a connection to the database using the server-side JDBC driver, you can either use "**jdbc:default:connection:**" as the URL for JDBC connection.
```
Comment (sungc-cubrid) [inline en/sql/jsp.rst:334]: I think it'll make more sense if the word "either" is deleted. When "either" is used, "or" will always follow in the sentence.

Comment (sungc-cubrid) [inline en/sql/jsp.rst]: I think it'll make more sense if the word "either" is deleted. When "either" is used, "or" will always follow in the sentence.

**Review by sungc-cubrid — CHANGES_REQUESTED** [review]
I think it'll make more sense if the word "either" is deleted. When "either" is used, "or" will always follow in the sentence.

File: en/sql/jsp.rst (hunk 1 above)
Comment (hgryoo) [inline en/sql/jsp.rst]:
```suggestion
To acquire a connection to the database using the server-side JDBC driver, you can use "**jdbc:default:connection:**" as the URL for JDBC connection.
```

Comment (hgryoo) [reply en/sql/jsp.rst]: @sungc-cubrid Thank you for your careful review! I've fixed it.

**Review by tw-kang — APPROVED** [review]

**Comment by hgryoo** [comment]
> I think it'll make more sense if the word "either" is deleted. When "either" is used, "or" will always follow in the sentence.

I've fixed your review at 6f394f3. Please change the status to "Approved" after checking it.

**Review by sungc-cubrid — APPROVED** [review]

**Pull request merged**